### ISSUE TITLE
The illustration appears to be different from the title

### ISSUE THREAD

**Issue by tnak160521** [NetAppDocs/ontap-apps-dbs#28]
### Page URL

https://docs.netapp.com/us-en/ontap-apps-dbs/vmware/vmware-vvols-protect.html

### Page title

Protecting vVols

### Summary

Even though the title is SnapCenter architecture, the figure is a diagram explaining the network configuration between storage and ESXi servers.

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by netapp-bingen** [comment]
Yes, it looks like the original image didn't make it through the conversion. I've got the original and will fix it on an internal branch for testing.
